### PR TITLE
feat: add subtitle and correct title style to blog overview page

### PR DESCRIPTION
**Pull request by feedm3**
> Feel free to merge this PR after approval

I aligned the title style and added the subtitle to the blog overview page:

Screenshot 2021-07-16 at 13 14 47

### CODE REVIEW

File: blog-posts/optimze-gatsby-urls.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ If there is a path without a trailing slash, you will now get a warning in the c
 trailing slash here, as the problem should be solved at the root - paths without a trailing slash will lead to potential
 follow-up problems.
 
-**Canonical**
+### Canonical
```
Comment (georgiee) [inline blog-posts/optimze-gatsby-urls.md:169]: you sneaky change😁

File: src/pages/blog.tsx [hunk 2]
```diff
@@ -85,6 +83,14 @@ const BlogPage: React.FC = () => {
         <GridItem>
           <BlogPageTitle>Blog</BlogPageTitle>
         </GridItem>
+        <GridItem md={8}>
+          <BlogPageSubTitl>
+            In our blog, our developers and designers talk about the latest
+            trends and know-how around tech and especially about our latest
+            learnings and insights on how to create incredibly good software for
+            the web.
```
Comment (georgiee) [inline src/pages/blog.tsx]: > create incredibly good software for the web.
might be true but that sounds incredible over the edge 😁

What about:
> Our developers and designers talk about the latest trends and know-how around tech, especially about our latest learnings and insights on creating good software for the web.

**Review by georgiee — CHANGES_REQUESTED** [review]
Greta thank you,
could you update the text a little bit?

File: src/pages/blog.tsx (hunk 2 above)
Comment (feedm3) [reply src/pages/blog.tsx]: Took the text from the design, but yes, maybe a bit too much 😄  Updated it to your suggestion

**Pull request merged**